### ISSUE TITLE
Generic plugin broken in Docker image

### ISSUE THREAD

**Issue by ljbs** [astefanutti/decktape#170]
Using the brand new Docker image of v2.9.0 (af5edd0), Puppeteer crashes on:

```
Error: Protocol error (Runtime.addBinding): 'Runtime.addBinding' wasn't found
```

Based on [Puppeteer#2922](https://github.com/GoogleChrome/puppeteer/issues/2922), this likely to be caused by an incompatibility of [Puppeteer 1.9.0](https://github.com/astefanutti/decktape/blob/af5edd060d13f18bdd83057d23ee2596ed8ed7c1/npm-shrinkwrap.json#L530) and the Chromium version that is fetched (through `apk`) while building the Docker image.

**Comment by astefanutti** [comment]
Thanks for the report. I'm unable to reproduce the issue locally, e.g.:
```
$ docker run --rm -t -v `pwd`:/slides astefanutti/decktape:2.9.0 http://remarkjs.com/ test.pdf
Loading page http://remarkjs.com/ ...
Loading page finished with status: 200
Remark JS plugin activated
Printing slide #19      (19/19) ...
Printed 19 slides
```

The Docker image installs Chromium 68, which seems to be working with Puppeteer v1.8.0 so far. I can revert back to using Puppeteer v1.8.0, though I wonder why it would work differently for me.

**Comment by ljbs** [comment]
Thanks for the swift reply! At first I thought it was a general issue, but indeed the example you're using above works fine for me as well. It doesn't, however, in `generic` mode (which I was also initially using):

```
$ node /decktape/decktape.js --chrome-path=chromium-browser --chrome-arg=--no-sandbox generic http://remarkjs.com/ test.pdf
```
(Just to be clear: I'm running it using the same image `astefanutti/decktape:2.9.0`, but explicitly calling it as I'm overriding the entry point to make it compatible with GitLab CI.)

Some logging in the console learns me the error occurs in [`exposeFunction()`](https://github.com/astefanutti/decktape/blob/76d8c03ff92273d9e6574330bdcc91a6ee95968a/plugins/generic.js#L56). I wanted to test with an earlier Puppeteer version (e.g. 1.8.0), but don't immediately manage to properly install it at this point - I might come back to it later.

**Comment by astefanutti** [comment]
Thanks for the details, that helps a lot!

I've tested it and it's been broken since Decktape v2.8.5 with the upgrade Puppeteer to version 1.6.0, which targets Chromium 69.0.3477.0, while the Docker image installs Chromium 68.

**Comment by astefanutti** [comment]
Fixed with 3dfea7bd0c850e7034b8fdbd5ab35614986bd783 and available in version `2.9.1`.
